concourse-chrome-driver/terraform/deployments/670214072732/codepipeline.tf

```diff
@@ -0,0 +1,119 @@
+resource "aws_codepipeline" "cd-chrome-driver" {
+  name     = var.pipeline_name
+  role_arn = data.aws_iam_role.pipeline_role.arn
+  tags     = merge(local.tags, { Name = "${var.pipeline_name}" })
+
+  artifact_store {
+    type     = "S3"
+    location = data.aws_s3_bucket.artifact_store.bucket
+  }
+
+  stage {
+    name = "Source"
+
+    action {
+      name             = "Source"
+      category         = "Source"
+      owner            = "AWS"
+      provider         = "CodeStarSourceConnection"
+      version          = "1"
+      output_artifacts = ["git_base_image"]
+      configuration = {
+        ConnectionArn    = "arn:aws:codestar-connections:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:connection/${var.codestar_connection_id}"
+        FullRepositoryId = "alphagov/cyber-security-concourse-base-image"
+        BranchName       = "CE-256-migrate-pipeline"
```

Comment:
Don't forget to change the branch name back

Reply:
done thanks 👍